### PR TITLE
[Trilinos] skip tests if solver is disabled

### PR DESCRIPTION
**Pull request by ddiezrod**
See title. Im sorry @philbucher but your name always comes out. : P

FYI @KratosMultiphysics/altair

### CODE REVIEW

**Comment by philbucher** [comment]
I did not add this on purpose as I thought disabling solvers would be the exception :D
Again I was wrong I see

=> can you please make it consistent and also do the same for Amesos & Aztec? Thx

File: applications/TrilinosApplication/tests/test_trilinos_linear_solvers.py [hunk 1]
```diff
@@ -379,6 +379,7 @@ def test_aztec_gmres(self):
             """)
 
 class TestMLLinearSolvers(TestLinearSolvers):
+    @KratosUnittest.skipIf(not hasattr(KratosMultiphysics.TrilinosApplication, 'MultiLevelSolver'), "MultiLevelSolver is not available.")
```
Comment (philbucher) [inline applications/TrilinosApplication/tests/test_trilinos_linear_solvers.py]:
```suggestion
    @KratosUnittest.skipIf(not hasattr(KratosMultiphysics.TrilinosApplication, 'MultiLevelSolver'), "MultiLevelSolver was explicitly disabled.")

```

Comment (ddiezrod) [reply applications/TrilinosApplication/tests/test_trilinos_linear_solvers.py]: done

**Comment by ddiezrod** [comment]
I added the skips to aztec too, It seems aesos tests are already configured to be skipped if solver is not available.

File: applications/TrilinosApplication/tests/test_trilinos_linear_solvers.py [hunk 2]
```diff
@@ -296,6 +296,7 @@ def test_amesos2_basker_2(self):
             """)
 
 class TestAztecLinearSolvers(TestLinearSolvers):
+    @KratosUnittest.skipIf(not hasattr(KratosMultiphysics.TrilinosApplication, 'AztecSolver'), "AztecSolver was explicitly disabled.")
```
Comment (philbucher) [inline applications/TrilinosApplication/tests/test_trilinos_linear_solvers.py]:
```suggestion
@KratosUnittest.skipUnless(hasattr(KratosMultiphysics.TrilinosApplication, 'AztecSolver'), "AztecSolver was explicitly disabled.")
class TestAztecLinearSolvers(TestLinearSolvers):

```
same can be done for the other tests, the Aztec and the ML (unlike the Amesos) have no options to configure so you can disable the entire test class

**Review by philbucher — COMMENTED** [review]
sorry one more comment :)

File: applications/TrilinosApplication/tests/test_trilinos_linear_solvers.py (hunk 2 above)
Comment (ddiezrod) [reply applications/TrilinosApplication/tests/test_trilinos_linear_solvers.py]: nice, done

**Review by philbucher — APPROVED** [review]
:+1:

**Comment by ddiezrod** [comment]
merging, thanks @philbucher

**Pull request merged**